src/main/kotlin/org/rust/lang/core/psi/ext/RsFunction.kt

```diff
@@ -73,7 +73,11 @@ val RsFunction.abiName: String?
         return stub?.abiName ?: abi?.stringLiteral?.text
     }
 
+/** Returns those function parameters that are not disabled by cfg attributes */
```

Comment:
I'd add some description to help contributors understand which one should be used in their case. Probably something like that
```suggestion
/**
 * Those function parameters that are not disabled by cfg attributes.
 *
 * Should be used in code analysis: name resolution, type inference, inspections, annotations, etc.
 */
```